apps/meteor/client/views/admin/moderation/ModerationConsoleTableRow.tsx

```diff
@@ -22,6 +22,13 @@ const ModerationConsoleTableRow = ({ report, onClick, isDesktopOrLarger }: Moder
 		return room.fname || room.name;
 	});
 
+	const normalizeMessage = () => {
+		if (message.startsWith('[ ](') && message.includes('\n')) {
```

Comment:
I wonder if it would not be better to have regex here to do the recognization instead. @ggazzo can you give your input here please?

Reply:
@csuadev if we're going to render the message, we should use the parser to render it properly. But since messages wasn't made to be render in a table I'm wondering if we should have this. Is it really valuable for the users? shouldn't only the context bar be responsible to show the message content since it has enough space to that? I think would be great to discuss it with the designers, since we do have a lot of corner cases rendering messages in a table.